qbft/instance.go

```diff
@@ -217,3 +220,41 @@ func (i *Instance) Decode(data []byte) error {
 func (i *Instance) CanProcessMessages() bool {
 	return !i.forceStop && int(i.State.Round) < CutoffRound
 }
+
+/////////////////////// JSON Marshalling for Tests ///////////////////////
```

Comment:
if it's for testing only:
1) why do we need it? which test depends on it?
2) let's separate it to a dedicated file

Reply:
1. It only prints `forceStop` when the value is `true`. All tests that have been forcedStopped depend on it.
2. done

There are some test that have nothing in their state change but `forceStop` field. If we don't print it we lose coverage.

In general our jsons are too verbose and hard to review. I would like to switch to an approach that we only print fields that change their default value.